packages/material-ui-styles/src/makeStyles/makeStyles.js

```diff
@@ -110,7 +109,8 @@ function attach({ state, theme, stylesOptions, stylesCreator, name }, props) {
       ...options,
     });
 
-    warning(props, 'Material-UI: props missing.');
+    // never true
+    // warning(props, 'Material-UI: props missing.');
```

Comment:
`useStyles()` is fairly common looking at all the TS issues. Supporting this makes sense since most styles aren't dynamic. I would however question why we need the default value anyway. Rather we should let this flow through and either let it crash with the usual null-pointer exception and rely on a typed language or throw a custom TypeError.